trin-execution/src/cli.rs

```diff
@@ -54,12 +56,16 @@ pub enum TrinExecutionSubCommands {
 
 #[derive(Args, Debug, Default, Clone, PartialEq)]
 pub struct ImportStateConfig {
+    #[arg(long, help = "The directory where to import the snapshot data.")]
+    pub data_dir: Option<PathBuf>,
     #[arg(long, help = "path to where the era2 state snapshot is located")]
     pub path_to_era2: PathBuf,
 }
 
 #[derive(Args, Debug, Default, Clone, PartialEq)]
 pub struct ExportStateConfig {
+    #[arg(long, help = "The directory from where to export the snapshot data.")]
+    pub data_dir: Option<PathBuf>,
```

Comment:
same comment about this, I think the ``TrinExecutionConfig` should be used I don't think we should add a param here as we already have one